### ISSUE TITLE
Evaluate and recommend Workshop topics

### ISSUE THREAD

**Issue by NarmathaBala** [microsoft/code-with-engineering-playbook#744]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by MagdaPaj** [comment]
At some point we talked about doing a workshop about IoT Edge observability. However, currently Azure platform doesn't have good observability solution. Our team had a meeting with the Edge PG and they are still working on defining it, but no specific ETA. But there is clearly a need for the Edge observability (at least I noticed it in manufacturing and automotive industry) so it would be good to help other dev crews.

We would need to decide what is the goal of the Edge observability workshop. Ideas:
1. **Do we want to make a recommendation about what to use?** Here we would need to work closely with the SA. There are some recommendations from the PG in the documentation, but it's based on [the Azure Sample code](https://github.com/Azure-Samples/iotedge-logging-and-monitoring-solution) and recently the PG admitted that they don't have resources to maintain and improve it and it's community driven effort. And that solution has quite a lot of problems. So at least from my perspective it is still not clear if we should keep recommending this solution, so it would be good to get feedback from SA.
2. **Do we want to present current state?** Our team had quite a lot of learnings, I'm happy to share it. And also collaborate with other teams/SA who has experience with it to get broader context (we only used metrics, logs, and log-based alerts, we didn't do anything with tracing, offline monitoring, and I'm not sure if that's available, host monitoring was very limited).
3. **Do we want to present customer challenges?** - I could talk about our customer challenges, and maybe collect feedback from other teams.

These are the options that I see, I'm happy to have more discussion about it, and it would be great to work together with the SA.
